<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseOfficeLocationWindow</class>
 <widget class="QMainWindow" name="chooseOfficeLocationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1954</width>
    <height>878</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(110, 204, 175);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1934</width>
         <height>815</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>1920</width>
         <height>815</height>
        </size>
       </property>
       <widget class="QGroupBox" name="choose_office_locationgroupBox">
        <property name="geometry">
         <rect>
          <x>860</x>
          <y>390</y>
          <width>441</width>
          <height>141</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(110, 204, 175);</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="select_location_pushButton">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>80</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(249, 249, 249);</string>
         </property>
         <property name="text">
          <string>Select Location</string>
         </property>
        </widget>
        <widget class="QComboBox" name="office_location_comboBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(249, 249, 249);</string>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Kernersville</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Winston-Salem</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mount Airy</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="choose_office_location_label">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>271</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Choose Office Location&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <zorder>choose_office_location_label</zorder>
        <zorder>office_location_comboBox</zorder>
        <zorder>select_location_pushButton</zorder>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>500</y>
          <width>381</width>
          <height>131</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(173, 231, 146);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="frame_4">
        <property name="geometry">
         <rect>
          <x>910</x>
          <y>440</y>
          <width>471</width>
          <height>151</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(173, 231, 146);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="frame_5">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>290</y>
          <width>161</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(243, 236, 176);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>330</y>
          <width>181</width>
          <height>261</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 77, 103);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>1050</x>
          <y>310</y>
          <width>381</width>
          <height>131</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(173, 231, 146);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="frame_10">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>0</y>
          <width>481</width>
          <height>151</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 77, 103);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>431</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Welcome!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_6">
        <property name="geometry">
         <rect>
          <x>1300</x>
          <y>520</y>
          <width>261</width>
          <height>121</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 77, 103);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="frame_11">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>100</y>
          <width>111</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(243, 236, 176);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="frame_8">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>440</y>
          <width>111</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(243, 236, 176);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="frame_12">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>-10</y>
          <width>161</width>
          <height>201</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(173, 231, 146);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>370</y>
          <width>181</width>
          <height>181</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 77, 103);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Images/Images/office.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_9">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>370</y>
          <width>481</width>
          <height>181</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 77, 103);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>4</number>
        </property>
        <property name="midLineWidth">
         <number>4</number>
        </property>
       </widget>
       <widget class="QFrame" name="frame_7">
        <property name="geometry">
         <rect>
          <x>1370</x>
          <y>250</y>
          <width>101</width>
          <height>101</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 77, 103);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <zorder>frame_6</zorder>
       <zorder>frame_2</zorder>
       <zorder>frame_5</zorder>
       <zorder>frame_4</zorder>
       <zorder>frame_9</zorder>
       <zorder>frame_12</zorder>
       <zorder>choose_office_locationgroupBox</zorder>
       <zorder>frame</zorder>
       <zorder>frame_3</zorder>
       <zorder>frame_10</zorder>
       <zorder>frame_11</zorder>
       <zorder>frame_8</zorder>
       <zorder>label</zorder>
       <zorder>frame_7</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1954</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
